api/api-pipeline-operations-v1/src/main/java/com/thoughtworks/go/apiv1/pipelineoperations/PipelineOperationsControllerV1.java

```diff
@@ -138,4 +153,16 @@ private PipelineScheduleOptions getScheduleOptions(Request req) {
         JsonReader jsonReader = gsonTransformer.jsonReaderFrom(req.body());
         return PipelineScheduleOptionsRepresenter.fromJSON(jsonReader);
     }
+
+    private Integer getCounterValue(Request request) {
+        try {
+            int counter = Integer.parseInt(request.params("pipeline_counter"), 10);
```

Comment:
Do we need to pass the radix? can't we simply use `Integer.parseInt(request.params("pipeline_counter"))`? where, by default, it converts the input to a signed decimal integer.

Reply:
No need to pass in the radix. Was just being cautious. Will remove the same.